src/build-data/cc/msvc.txt

```diff
@@ -48,7 +48,7 @@ default-debug -> "$(LINKER) /DLL /DEBUG"
 
 <binary_link_commands>
 default       -> "$(LINKER)"
-default-debug -> "$(LINKER) /DEBUG"
```

Comment:
I added `/DEBUG` in order to be able to run the CLI and tests in the Qt Creator debugger. Having different filenames sounds more solid to me.